deploy/virtual-cluster-components-manifest-cm.yaml

```diff
@@ -4,6 +4,7 @@ data:
     [
       {"name": "kube-proxy", "path": "/kosmos/manifest/kube-proxy/*.yaml"},
       {"name": "calico", "path": "/kosmos/manifest/calico/*.yaml"},
+      {"name": "keepalived", "path": "/kosmos/manifest/keepalived/*.yaml"},
     ]
```

Comment:
It is necessary to emphasize in the deployment document or create a separate chapter to introduce how to deploy VIP.

Reply:
add simple quick start guide in conf